src/Platform/Microsoft.Testing.Platform/Telemetry/Activities.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.Telemetry;
```

Comment:
Do we want the namespace to be "Telemetry" or "OpenTelemetry"?

Reply:
I see that as a telemetry but I don't mind or the other.